edx_solutions_api_integration/users/views.py

```diff
@@ -748,6 +748,10 @@ def post(self, request, user_id):  # pylint: disable=R0915
             existing_user.save()
             update_user_password_hash = existing_user.password
 
+            # clear failed login attempts counters, if applicable
+            if LoginFailures.is_feature_enabled():
```

Comment:
I think it's better if we only do that if user is in fact locked out to be on safer side. so
```suggestion
            if LoginFailures.is_feature_enabled() and LoginFailures.is_user_locked_out():
```

Reply:
Also add unit tests for your changes

Reply:
We don't have any test cases for UsersDetail method or any related method. So, to write the test case for it means to cover the whole method.